### PR TITLE
Update diagrams and README.

### PR DESCRIPTION
**Pull request by maruiz93**
Upgrade the arch diagram to v0.3
Remove the deprecated proposed version
of the DB model

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -7,14 +7,14 @@ This project was inspired by [Tekton Results](https://github.com/tektoncd/result
...
+- A deployer
+- One or more APIServerSources that send cloud events to a sink
+- A sink that receives the cloud events and write the resource information in a DB
+- A REST API server that facilitates the retrieval of cluster data.
+- A CLI that queries both the KubeArchive REST API and the k8s API to expose the resources
 
 ## Database Model
 
-![kubearchive database model](/profile/database-model.png)
+- WIP. Check [this Miro board](https://miro.com/app/board/uXjVNjKPncc=/) for more information.
```
Comment (adambkaplan) [inline profile/README.md:20]: Ideally we can make this representation public in the future. This MIRO currently can only be accessed by Red Hatters.

**Review by adambkaplan — APPROVED** [review]
LGTM

**Review by adambkaplan — APPROVED** [review]

File: profile/README.md (hunk 1 above)
Comment (maruiz93) [reply profile/README.md:20]: We'll probably also have a diagram soon to expose in here.

**Review by skoved — APPROVED** [review]
:+1:

File: profile/README.md [hunk 2]
```diff
@@ -7,14 +7,14 @@ This project was inspired by [Tekton Results](https://github.com/tektoncd/result
...
 
-- An operator that manages the system as a whole.
-- An archive controller that reads data from the cluster.
-- An archive service that receives data to archive, and stores it in a database.
-- (future) A log storage backend.
-- (future) A REST API server that facilitates the storage and retrieval of cluster data.
+- A deployer
+- One or more APIServerSources that send cloud events to a sink
+- A sink that receives the cloud events and write the resource information in a DB
+- A REST API server that facilitates the retrieval of cluster data.
```
Comment (beerparty) [inline profile/README.md]: None of the other bullet points have end punctuation, so for consistency you should remove the period at the end of this sentence.

```suggestion
- A REST API server that facilitates the retrieval of cluster data
```

**Pull request merged**